core/tests/ts-integration/tests/ether.test.ts

```diff
@@ -50,16 +54,46 @@ describe('ETH token checks', () => {
             gasPrice: await gasPrice
         });
 
+        // console.log('balances before deposit: ');
+        // console.log('alice.getBalance(): ', await alice.getBalance());
+        // console.log('alice.getBalanceL1(): ', await alice.getBalanceL1());
+        // console.log('base token address: ', process.env.CONTRACTS_BASE_TOKEN_ADDR);
+        // console.log('alice.getBalance(NATIVE TOKEN): ', await alice.getBalance(process.env.CONTRACTS_BASE_TOKEN_ADDR));
+        // console.log(
+        //     'alice.getBalanceL1(NATIVE TOKEN): ',
+        //     await alice.getBalanceL1(process.env.CONTRACTS_BASE_TOKEN_ADDR)
+        // );
+
         const depositOp = alice.deposit({
             token: ETH_ADDRESS,
             amount,
             gasPerPubdataByte,
             l2GasLimit,
+            approveBaseERC20: true,
+            approveERC20: true,
+            approveBaseOverrides: {
+                gasPrice
+            },
             overrides: {
                 gasPrice
             }
         });
-        await expect(depositOp).toBeAccepted([l2ethBalanceChange]);
+
+        if (isETHBasedChain) {
+            await expect(depositOp).toBeAccepted([l2ethBalanceChange]);
+        } else {
+            await expect(depositOp).toBeReverted(); // ToDo: after server fix has to be removed
+        }
+
+        await sleep(8);
+        // console.log('balances after deposit: ');
+        // console.log('alice.getBalance(): ', await alice.getBalance());
```

Comment:
I think you forgot to delete these comments, if however you want to print them then use the debugging reporter I think ( but not console logs)